docs/src/Experimental/intro.md

```diff
@@ -21,7 +21,7 @@ brought up to Oscar standard.
 
 ## Structure
 To look at a sample for the structure for a new project in `experimental` have
-a look at `experimental/PlaneCurve`. The general structure is
+a look at `experimental/{FTheoryTools, PlaneCurve}`. The general structure is
```

Comment:
I would not change this. Or is the plan to keep adding each experiment that gets converted? That's not helpful.

If you want to be prepared for a future in which `experimental/PlaneCurve` is gone, one could describe it generically: "look at subdirectories of `experimental` which contain a `src` subdirectory" or so.

Reply:
The plan was just to provide more than one example for now, especially since FTheoryTools is a little bit more involved. I plan to remove these specific examples once these procedures have been established.